scipy/stats/morestats.py

```diff
@@ -581,9 +582,8 @@ def probplot(x, sparams=(), dist='norm', fit=True, plot=None, rvalue=False):
 
     """
     x = np.asarray(x)
-    _perform_fit = fit or (plot is not None)
```

Comment:
Clearly plotting the least-squares fit regardless of `fit` was the intended behavior, but I think @jackbenn's suggestion in gh-13769 makes more sense.